scripts/migrate-vfolder-structure.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import os
+
+
+def main():
+    input_path = sys.argv[1]  # full path for vroot/local
+    output_path = sys.argv[2]  # full path for volume directory. ex. vfs/
+    print(input_path, output_path)
+
+    subfolders = [f.path for f in os.scandir(input_path) if f.is_dir()]
+    print("Subfolders ", subfolders)
+    for folder in subfolders:
+
+        folder = folder.split("/")[-1]
+        header_dir = folder[0:2]
+        second_lvl_dir = folder[2:4]
+        rest_dir = folder[4:]
+
+        list_of_files = os.listdir(input_path + "/" + folder)
+        print(list_of_files)
+        print("list of files ", list_of_files)
+        try:
+            if not os.path.isdir(output_path + "/" + header_dir):
+                os.mkdir(output_path + "/" + header_dir)
+            if not os.path.isdir(output_path + "/" + header_dir + "/" + second_lvl_dir):
+                os.mkdir(output_path + "/" + header_dir + "/" + second_lvl_dir)
+            if not os.path.isdir(output_path + "/" + header_dir + "/" + second_lvl_dir + "/" + rest_dir):
+                os.mkdir(output_path + "/" + header_dir + "/" + second_lvl_dir + "/" + rest_dir)
```

Comment:
You could do this part with one line: `Path(output_path / header_dir / second_lvl_dir).mkdir(parents=True, exist_ok=True)`